app/Http/Controllers/ExportController.php

```diff
@@ -33,7 +33,7 @@ public function __construct(ExportService $export)
     public function show($campaignId)
     {
         // Run the query
-        $signups = Signup::whereNotNull('details')->where('campaign_id', $campaignId)->get();
+        $signups = Signup::whereNull('details')->where('campaign_id', $campaignId)->get();
```

Comment:
For most campaigns, this will cause the export to include all signups for the campaign. Should we consider grabbing specific details here for now?

Reply:
oh wait that still wouldn't help because the details we want are `null` hmmmm

Reply:
hmm well it's weird cause for this campaign, we only want to export the `NULL` values. But we only want to allow that option if it was offered. I considered trying to update PN to send all opt-ins with some details but that seemed like a bit of a larger lift since we already worked the logic around opting-out. So i'm fine for now, if we just export all signups everywhere, but I would love if we only offered this option on the campaigns where that makes sense.

Reply:
@sbsmith86 Yeah I started confusing myself with double negatives haha. I'm on board with this for now and eventually only showing the exports where relevant!

Reply:
cool. Can I get the final approval? I will add a card for re-working this

Reply:
@sbsmith86 whoops sorry! of course!